libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -168,6 +178,17 @@ constructor(
             .createMapboxTimer(navigationOptions.fasterRouteDetectorInterval) {
                 requestFasterRoute()
             }
+        ifNonNull(accessToken) { token ->
+            MapboxMetricsReporter.init(context, token, obtainUserAgent())
```

Comment:
We should also toggle logging 👀 legacy `MapboxNavigation` https://github.com/mapbox/mapbox-navigation-android/blob/46dd7c6fb56fcb34a5b9da5341894441bf05e547/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigation.java#L1094